src/lib_ccx/ocr.c

```diff
@@ -810,14 +803,32 @@ int compare_rect_by_ypos(const void*p1, const void *p2, void*arg)
 
 void add_ocrtext2str(char *dest, char *src, const char *crlf, unsigned crlf_length)
 {
+	char *line_scan;
+	BOOL should_break = FALSE;
```

Comment:
I'm not a big fan of BOOL - in C bools are int, 0 is FALSE and everything else is TRUE...